src/services/passport.ts

```diff
@@ -0,0 +1,86 @@
+import axios, { AxiosInstance } from "axios";
+import { PASSPORT_API_KEY } from "../../config.json";
+import {
+  LinkAccountBody,
+  LinkedAccountsResponse,
+  LinkedAddressesResponse,
+} from "../types/passport";
+
+const baseURL = "https://galactic-passport-server.herokuapp.com/v1";
+
+class PassportAPI {
+  passportClient: AxiosInstance;
+
+  constructor() {
+    this.passportClient = axios.create({
+      baseURL,
+      headers: {
+        "X-GP-API-KEY": PASSPORT_API_KEY,
+        "Content-Type": "application/json",
+      },
+    });
+  }
+
+  // get requests
+
+  // get wallets of discord id
+  getWalletsByDiscordId = async (discordId: string): Promise<string[]> => {
+    const res = (
+      await this.passportClient.get(
+        `/linked_addresses?platform_ids=discord_${discordId}&onlyIfPrimaryAccount=true`
+      )
+    ).data as LinkedAddressesResponse;
+
+    const wallets = res[discordId].wallets;
+
+    if (wallets == undefined) {
+      throw new Error("Wallets returned are undefined");
+    }
+
+    return wallets.map((walletObj) => walletObj.address);
+  };
+
+  // get discord id of wallet
+  getDiscordIdByWallet = async (address: string): Promise<string> => {
+    const res = (
+      await this.passportClient.get(
+        `/linked_accounts?platforms=discord&addresses=${address}`
```

Comment:
Hmmm this `discordId` isn't unique. Need to grab the primary Discord here... For ex:
`/primary_accounts/?platforms=discord&addresses=${address}`

Reply:
Response looks like:
```json
{
  "addr1": {
    "primaryDiscord": "someDiscordId",
  }
}
```

Reply:
```javascript
  // get discord id of wallet
  getDiscordIdByWallet = async (address: string): Promise<string> => {
    const res = (
      await this.passportClient.get(
        `/primary_accounts?platforms=discord&addresses=${address}`
      )
    ).data as PrimaryAccountsResponse;

    if (
      res[address] == undefined ||
      res[address]["primaryDiscord"] == undefined
    ) {
      throw new Error("Can't find discord ID");
    }

    const discordId = res[address]["primaryDiscord"];
    return discordId;
  };
```